readme.md

```diff
@@ -104,6 +119,10 @@ Can also be provided as the environment variable `BUCKET_NAME`.
 An optional folder to stream log files to. Takes a path string,
 eg: "my/subfolder" or "nested".
 
+#### folder
```

Comment:
this should be `#### tags` presumably :)

Reply:
You're right. I copied the folders section but forgot to change the section title.